modules/navier_stokes/doc/content/source/fvbcs/WCNSFVMomentumFluxDirichletBC.md

```diff
@@ -0,0 +1,21 @@
+# WCNSFVMomentumFluxDirichletBC
+
+!syntax description /FVBCs/WCNSFVMomentumFluxDirichletBC
+
+This boundary condition is similar to [WCNSFVMassFluxBC.md], except that it enforced the mass
+flow rate provided by the user as a Dirichlet boundary conditions. Hence, using this boundary
+condition the mass flow rate at the inlet faces will always be the one specified by the user.
```

Comment:
The mass flow rate was respected in the mass equation (by the solver), the issue was that the `VolumetricFlowRate` PP did not use the velocity computed from the flux value on the face, rather an extrapolated value based on the velocity field. 

A few months ago, I added another postprocessor which actually uses the mass flux provided by the user:
https://mooseframework.inl.gov/source/postprocessors/SideFVFluxBCIntegral.html

which should give the exact value back. Could you mention these details in the documentation ?

At the same time, I am fine having a Dirichlet BC for this if that makes the DO coupling much easier.